Mathlib/CategoryTheory/Monoidal/Mon_.lean

```diff
@@ -203,6 +203,9 @@ instance {A B : Mon_ C} (f : A ⟶ B) [e : IsIso ((forget C).map f)] : IsIso f.h
 instance : (forget C).ReflectsIsomorphisms where
   reflects f e := ⟨⟨.mk' (inv f.hom), by aesop_cat⟩⟩
 
+instance {M N : Mon_ C} {f : M ⟶ N} [IsIso f] : IsIso f.hom :=
+  inferInstanceAs <| IsIso <| (forget C).map f
+
```

Comment:
Does the instance on line 199 help with this? Alternatively, now we have "two ways" of proving `IsIso f.hom`, is it clear that we need both?